.github/ISSUE_TEMPLATE/Platform _Epic_Template.md

```diff
@@ -0,0 +1,24 @@
+---
+Name: Epic template
+About: Template for creating  platform epics
+Title: "[EPIC]"
+Labels: epic platform
+
+---
+
+## Outcome/Objective ##
```

Comment:
Perhaps we don't need to do this now, but I think we really do need some comments for each of these headers explaining what content should go under the header and how it should be formatted. The reason I say we can wait is because perhaps we need to figure it out as we go.